tldw_Server_API/app/core/Setup/audio_profile_service.py

```
@@ -0,0 +1,258 @@
+                    resource_profile=resource_profile,
+                    selection_key=build_audio_selection_key(bundle.bundle_id, resource_profile, bundle.catalog_version),
+                    confidence=confidence,
+                    reasons=reasons,
+                    score=score,
+                )
+            )
+
+    return sorted(recommendations, key=lambda item: item.score, reverse=True)
+
+
+def _score_bundle_profile(
```

Ruff (0.15.6)

[warning] Too many branches (20 > 12)

(PLR0912)